frontend/src/components/accounts/AccountsList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import * as A from "../../libraries/explorer-wamp/accounts";
```

Comment:
For what it worth, I used `* as T` where `T` states for "types" (not "transactions") ^_^